test/test_core.py

```diff
@@ -3758,6 +3758,20 @@ def test_dlfcn_asyncify(self):
       '''
     self.do_run(src, 'before sleep\nafter sleep\n42\n')
 
+  @requires_jspi
+  @needs_dylink
+  def test_dlfcn_jspi(self):
+    side_flags = ['-sSIDE_MODULE']
+    if self.get_setting('MEMORY64'):
+      side_flags.append('-sMEMORY64')
+    self.set_setting('JSPI')
+    self.run_process([
+      EMCC,
+      '-o', 'side.so',
+      test_file('core/test_dlfcn_jspi_side.c'),
+      *side_flags])
```

Comment:
Maybe better do put `-sSIDE_MODULE` here and then add `+ self.get_emcc_args()` to the end of the command.  That way to the global flags (such as `-sMEMORY64) will be added there.